docs/serving/using-auto-tls.md

```diff
@@ -78,170 +92,213 @@ To enable support for automatic TLS certificate provisioning in Knative:
           email: myemail@gmail.com
           privateKeySecretRef:
             # Set privateKeySecretRef to any unused secret name.
-            name: letsencrypt-issuer
-          dns01:
-            providers:
-            - name: cloud-dns-provider
-              clouddns:
-                # Set this to your GCP project-id
-                project: $PROJECT_ID
-                # Set this to the secret that we publish our service account key
-                # in the previous step.
-                serviceAccountSecretRef:
-                  name: cloud-dns-key
-                  key: key.json
+            name: letsencrypt-dns-issuer
+          solvers:
+          - dns01:
+            clouddns:
+              # Set this to your GCP project-id
+              project: $PROJECT_ID
+              # Set this to the secret that we publish our service account key
+              # in the previous step.
+              serviceAccountSecretRef:
+                name: cloud-dns-key
+                key: key.json
       ```
 
-   1. Add your `ClusterIssuer` configuration to your Knative cluster by running
-      the following commands, where `<filename>` is the name of the file that
-      you created:
+1.  ClusterIssuer for HTTP01 challenge
```

Comment:
Again, this seems confusing structured this way - make this a heading instead with numbered steps on how to create the ClusterIssuer?